### PR TITLE
Add missing first datetime functions to daily active addresses

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Summary

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

File: lib/sanbase/clickhouse/daily_active_addresses.ex [hunk 1]
```diff
@@ -26,8 +39,14 @@ defmodule Sanbase.Clickhouse.DailyActiveAddresses do
...
   end
 
+  def average_active_addresses(btc, from, to, interval)
+      when is_binary(btc) and btc in @bitcoin do
+    Bitcoin.average_active_addresses(from, to, interval)
+  end
+
   def average_active_addresses(contract, from, to, interval) when is_binary(contract) do
     Erc20.average_active_addresses(contract, from, to, interval)
+    |> IO.inspect(label: "AVG ADDRESSES FOR #{contract}")
```
Comment (kalkov) [inline lib/sanbase/clickhouse/daily_active_addresses.ex]: Forgotten inspect

Comment (tspenov) [inline lib/sanbase/clickhouse/daily_active_addresses.ex]: Inspect left

Comment (IvanIvanoff) [reply lib/sanbase/clickhouse/daily_active_addresses.ex]: fixed

Comment (IvanIvanoff) [reply lib/sanbase/clickhouse/daily_active_addresses.ex]: yep, @kalkov also reported and it's fixed now

**Pull request merged**